pkg/resources/instance/resource.go

```diff
@@ -207,6 +217,21 @@ func Resource() *schema.Resource {
 	}
 }
 
+func findInstanceType(ctx context.Context, c *v3.Client, x string) (*v3.InstanceType, error) {
```

Comment:
There is `FindInstanceType` in v3 which should do the same thing (@pierre-emmanuelJ to confirm):
https://github.com/exoscale/egoscale/blob/master/v3/operations.go#L8812C36-L8812C52

Reply:
Ah yes good catch @kobajagi the one I generated last week :)